sympy/utilities/lambdify.py

```diff
@@ -450,6 +452,19 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def print_dummify(expr):
+    """
+    Returns the value of dummify to be used. This function either returns True or False.
+    >>> from sympy.utilities.lambdify import print_dummify
+    >>> from sympy.abc import x, y
+    >>> print_dummify(x + y**2)
+    True
+
+    """
+    valid_identifier_regex = re.compile('^[^\d\W]\w*\Z', re.UNICODE)
+    result = valid_identifier_regex.match(str(expr))
```

Comment:
@asmeurer , by passing in printer do you mean using `_print`. Could you please clarify. Also what is the problem is using a typecast in `str()` ?